promql/parse.go

```diff
@@ -609,7 +624,7 @@ func (p *parser) rangeSelector(vs *VectorSelector) *MatrixSelector {
 	const ctx = "matrix selector"
```

Comment:
@fabxc @juliusv In the documentation we call that actually a `range selector`. Or am I missing something?